app/concepts/reservations/use_cases/create_online.rb

```diff
@@ -0,0 +1,33 @@
+module Reservations
+  module UseCases
+    class CreateOnline  < Reservations::UseCases::CreateReservationWithTickets
+      attr_reader :repository
+
+      def initialize(repository: Reservations::Repository.new)
+        @repository = repository
+      end
```

Comment:
As you are not changing anything here compared to the parent class, this method is not needed, if child does not override method that method will be called on parent, so here if `initialize` will not be defined `initialize` from `Reservations::UseCases::CreateReservationWithTickets` will be used